bin/check-postgres-replication.rb

```diff
@@ -103,8 +104,9 @@ class CheckPostgresReplicationStatus < Sensu::Plugin::Check::CLI
   def run
     ssl_mode = config[:ssl] ? 'require' : 'prefer'
 
-    critical 'Slave host is undefined' unless config[:slave_host] && !['localhost', '127.0.0.1'].include?(config[:slave_host])
-    critical 'Master and slave cannot be the same host' unless config[:master_host] != config[:slave_host]
+    critical 'Master cannot be localhost' if ['localhost', '127.0.0.1'].include?(config[:master_host])
```

Comment:
do we still need to make the localhost checks still since we now require both to be specified?

Reply:
:+1: makes sense. I will remove.